content/chainguard/chainguard-images/retrieve-image-sboms.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "How to Retrieve SBOMs for Chainguard Images"
+linktitle: "Retrieve an Image's SBOM"
+type: "article"
+description: "A brief tutorial on how to use Cosign to retrieve Chainguard Image SBOMs."
+date: 2023-11-17T11:07:52+02:00
+lastmod: 2023-11-17T11:07:52+02:00
+draft: false
+tags: ["Conceptual", "Chainguard Images", "SBOM"]
+images: []
+menu:
+  docs:
+    parent: "chainguard-images"
+weight: 710
+toc: true
+---
+
+
+Chainguard Images contain only the minimum number of packages needed to use the software they contain. The purpose of this is to reduce the image's attack surface and minimize the risk that CVEs will impact software that depends on these container images. 
+
+Even though they contain the minimum number of packages, there may come a time when you want to know exactly what's running inside of a certain Chainguard Image. For this reason, we include a signed SBOM (also known as an *attestation*) with each image.
```

Comment:
```suggestion
Even though they contain the minimum number of packages, there may come a time when you want to know exactly what's running inside of a certain Chainguard Image. For this reason, we include a signed SBOM with each image in the form of a [software attestation](https://slsa.dev/attestation-model).
```

Reply:
I think some people could read this and understand that an SBOM is an attestation, when in fact we offer our sboms as a slsa attestation but there are other types of attestations.